job/replay_manager.go

```diff
@@ -156,6 +158,25 @@ func (m *Manager) GetReplay(replayUUID uuid.UUID) (models.ReplaySpec, error) {
 	return m.replaySpecRepoFac.New().GetByID(replayUUID)
 }
 
+// GetReplayList using Project ID
+func (m *Manager) GetReplayList(projectUUID uuid.UUID) ([]models.ReplaySpec, error) {
+	replays, err := m.replaySpecRepoFac.New().GetByProjectID(projectUUID)
+	if err != nil {
+		if err == store.ErrResourceNotFound {
+			return []models.ReplaySpec{}, nil
+		}
+		return []models.ReplaySpec{}, err
+	}
+
+	var recentReplays []models.ReplaySpec
+	for _, replay := range replays {
+		if replay.CreatedAt.After(time.Now().Add(replayListWindow)) {
```

Comment:
how are you testing this function if you are using `time.Now()` here?

Reply:
testing it this way: https://github.com/odpf/optimus/blob/8c19f89adf3038f7c787a74bdf1650626d584fb2/job/replay_manager_test.go#L374

since this function is not too time sensitive (have wide time window: 90 days), i simply use the current time of the test, and exclude the replay that aged 100 days. let me know if this is not proper.

Reply:
but this test will start to fail after few 3 months because your replay specs are hardcoded with createdAt right? 😄

Reply:
the createdAt replaySpecs in test has value of today minus several days so it wont pass after 3 months 😄 but i'm also fine if the starting window being passed as an argument is better and cleaner

Reply:
oh got it, so even the mocked replays are using `time.Now` on createdAt then I think it's fine. We can leave it like this. 
Also, we should use `UTC` with `Now()`.

Reply:
got it. have updated this